blog/20131108-debian-mate-packaging-team.md

```diff
@@ -8,21 +8,21 @@
 .. author: Stefano Karapetsas
 -->
 
-![Debian](http://www.debian.org/logos/openlogo-nd-100.png)
+![Debian](https://www.debian.org/logos/openlogo-nd-100.png)
 
 The MATE Team is very happy to say hello to the new Debian MATE Packaging
-Team, that is [working hard](http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=708385#31)
+Team, that is [working hard](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=708385#31)
 to get MATE included into the next release of Debian.
 
 First packages are already in the repositories and there are many others
-in [ftp-master NEW queue](http://ftp-master.debian.org/new.html).
+in [ftp-master NEW queue](https://ftp-master.debian.org/new.html).
 
 A big thanks to:
 
-  * [John Paul Adrian Glaubitz](http://users.physik.fu-berlin.de/~glaubitz/)
-  * [Mike Gabriel](http://sunweavers.net/blog/)
-  * [Vangelis Mouhtsis](http://gnugr-blog.info/)
-  * [Kees Cook](http://outflux.net/)
+  * [John Paul Adrian Glaubitz](https://users.physik.fu-berlin.de/~glaubitz/)
+  * [Mike Gabriel](https://sunweavers.net/blog/)
+  * [Vangelis Mouhtsis](https://gnugr-blog.info/)
```

Comment:
http://gnugr-blog.info works

https://gnugr-blog.info doesn't work

tested with firefox 68.2.0esr and chromium 76.0.3809.100

Reply:
http://gnugr-blog.info/ looks like a porn blog, i will delete the link

Reply:
Done